### DISCUSSION TITLE
No wifi password

### DISCUSSION THREAD

**Discussion by Patounet58** [roleoroleo/yi-hack-Allwinner-v2#875] [General]
Good morning,
No way to use the camera for 1 month, I don't use a wifi password and I systematically get the error wifi password message.
An idea to solve this problem (without imposing a wifi password)
Thanks and good day.

**Comment by brocciu** [comment]
Same problem here, the cam is useless. 
Is it possible to use only the hack ?

**Comment by roleoroleo** [comment]
Please, explain me better.
